drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -349,15 +351,23 @@ function DragAndDropTemplates(configuration) {
         return h(
             popupSelector, {style: {display: have_messages ? 'block' : 'none'} },
             [
-                h('div.close.icon-remove-sign.fa-times-circle'),
+                h(
+                    'button.unbutton.close.icon-remove-sign.fa-times-circle',
```

Comment:
@cahrens Unfortunately for some reason I was not able to find in 15 minutes, removing `fa-times-circle` from the button makes it ignore the clicks. Not removing it obviously results in double icon.

It's 9pm my timezone, so I won't be able to address that, sorry.